features/support/pages/admin.rb

```diff
@@ -0,0 +1,6 @@
+require_relative "admin/moderator_action_form_section"
+require_relative "admin/moderator_action_section"
+require_relative "admin/pending_project_section"
+
+require_relative "admin/pending_projects_page"
+require_relative "admin/moderator_actions_page"
```

Comment:
The PageObject/Section pattern seems to encourage creating a bunch of different tiny classes. I'm generally in favor of keeping them all in the same file; but most people seem to prefer single file per class. Thoughts @bhaibel ?